mirdata/datasets/billboard.py

```diff
@@ -0,0 +1,470 @@
+"""McGill Billboard Dataset Loader
+
+.. admonition:: Dataset Info
+    :class: dropdown
+
+    The McGill Billboard dataset includes annotations and audio features corresponding to 890 slots from a random sample of Billboard chart slots.
+    It also includes metadata like Billboard chart date, peak rank, artist name, etc.
+    Details can be found at https://ddmal.music.mcgill.ca/research/The_McGill_Billboard_Project_(Chord_Analysis_Dataset)
+"""
+
+import csv
+import logging
+import os
+import shutil
+import re
+from typing import BinaryIO, TextIO, Optional, Tuple
+
+import librosa
+import numpy as np
+
+from mirdata import download_utils
+from mirdata import jams_utils
+from mirdata import core
+from mirdata import annotations
+from mirdata import io
+
+BIBTEX = """
+@inproceedings{burgoyne_billboard,
+author = {Burgoyne, John Ashley and Wild, Jonathan and Fujinaga, Ichiro},
+year = {2011},
+title = {An {Expert} {Ground} {Truth} {Set} for {Audio} {Chord} {Recognition} and {Music} {Analysis}},
+booktitle={Proceedings of the 12th International Society for Music Information Retrieval Conference, ISMIR}
+}
+
+@phdthesis{phdthesis,
+  author       = {Burgoyne, John Ashley}, 
+  title        = {Stochastic {Processes} and {Database}-{Driven} {Musicology}},
+  school       = {McGill University, Montréal, Québec},
+  year         = 2012,
+}
+"""
+
+REMOTES = {
+    "index": download_utils.RemoteFileMetadata(
```

Comment:
nit: to not confuse this with the file index, could this be called something else like `metadata`?

Reply:
fixed in [0567be1a4630c922f649816524008e14a2dca04e](https://github.com/Spijkervet/mirdata/commit/0567be1a4630c922f649816524008e14a2dca04e)